src/utils/date.test.ts

```diff
@@ -0,0 +1,132 @@
+import MockDate from "mockdate";
+import {
+  getDateByTimeString,
+  getDifferenceBetweenDate,
+  getCalendar,
+  getFormattedDate
+} from "./date";
+
+beforeAll(() => {
+  MockDate.set("2019-05-18");
+});
+
+afterAll(() => {
+  MockDate.reset();
+});
+
+describe("Testing date utility", () => {
```

Comment:
please re-check the linings after every test block. either have empty line after the blocks or not